### ISSUE TITLE
Hyprland is broken

### ISSUE THREAD

**Issue by Froggo8311** [hyprwm/Hyprland#1654]
The other day, I updated Hyprland from the AUR package. I didn't notice any changes until now, as I remembered I had been running an older build of hyprland that I had placed on my PATH. I removed it (bad idea lol), and I found 
out that the build installed from the AUR just segfaults when I try to launch Hyprland. I cloned and built Hyprland myself, only to find that it is also broken. When I try to open kitty with Super+Q when using the auto-generated
config (as mine caused it to immediately crash without error), that just crashes Hyprland. I'm not really sure what to do as of right now, other than finding an older commit that builds properly.

**Comment by Froggo8311** [comment]
I was able to download the latest release, and that is stable for now. Guess I'll hold off on updating next time 😅

**Comment by kumarmo2** [comment]
> I was able to download the latest release, and that is stable for now. Guess I'll hold off on updating next time sweat_smile

Hey When I took the latest release tag, I was getting build errors. Only after I took the latest commit from main branch, I was able to build succesfully. 

Just wanted to make sure if you also got the build error in case you also took the latest release only. If that is the case I was thinking of creating a new issue for a new release.

**Comment by vaxerski** [comment]
logs? crash logs? Saying "it crashes" means nothing.

**Comment by Froggo8311** [comment]
@vaxerski Strange. I downloaded the v0.22.0beta release binaries, lol.

Also, I forgot to mention, but there was zero logs. None. There was the usual notices about GL or whatever, but there we no errors. It just silently closes.

**Comment by vaxerski** [comment]
hyprland logs. Please read the issue guidelines before posting.

**Comment by Froggo8311** [comment]
There was nothing. Unless the logs stored in /tmp aren't always placed in STDOUT, then there was nothing related to the issue in the logs.

**Comment by vaxerski** [comment]
Just because you don't see something doesn't mean it's unrelated.

**Comment by Froggo8311** [comment]
Interesting. No, I've got an Intel intigrated GPU

**Comment by Froggo8311** [comment]
So weird. I just cloned & compiled, and everything is fine now. The screen stopped flickering and its not crashing when I try to open kitty. My computer needs to stop doing weird shit...

I literally am not making this up lol
